apps/web/screens/ServiceWeb/SubPage/SubPage.tsx

```diff
@@ -64,16 +73,26 @@ const SubPage: Screen<SubPageProps> = ({
   questionSlug,
   namespace,
   organizationNamespace,
+  singleSupportCategory,
+  locale,
 }) => {
   const Router = useRouter()
   const n = useNamespace(namespace)
   const o = useNamespace(organizationNamespace)
   const { linkResolver } = useLinkResolver()
+  useContentfulId(
+    organization.id,
+    singleSupportCategory?.id,
+    singleSupportQNA?.id,
+  )
+  useLocalLinkTypeResolver()
+
+  console.log('singlesupportqna', singleSupportQNA)
```

Comment:
:video_game: :wood:

Reply:
Oh my, nice find

Reply:
Will I have to ask people for another review if I commit now? That's what I'm worried about. If the release train goes at 10 'o clock

Reply:
I don't think so